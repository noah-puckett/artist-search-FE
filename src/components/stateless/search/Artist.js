import React from 'react';
import PropTypes from 'prop-types';
// import { Link } from 'react-router-dom';

function Artist({ name }) {
    return (<>
        {/* <Link to={`/${id}/${name}`} /> */}
        <h3>{name}</h3>
    </>
    );
}

Artist.propTypes = {
    name: PropTypes.string.isRequired,
    // id: PropTypes.string.isRequired
};

export default Artist;
