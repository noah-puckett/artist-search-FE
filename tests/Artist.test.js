import React from 'react';
import { shallow } from 'enzyme';
import Artist from '../src/components/stateless/search/Artist';

describe('Artist component', () => {
    it('renders Artist', () => {
        const wrapper = shallow(<Artist artists={[]} name='hello' />);
        expect(wrapper).toMatchSnapshot();
    });
});
