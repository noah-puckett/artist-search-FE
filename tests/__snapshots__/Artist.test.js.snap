// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist component renders Artist 1`] = `
<Fragment>
  <h3>
    hello
  </h3>
</Fragment>
`;
